internal/lm/nvml.go

```diff
@@ -101,11 +101,21 @@ func newVersionLabeler(manager resource.Manager) (Labeler, error) {
 	}
 
 	labels := Labels{
+		// Deprecated labels
 		"nvidia.com/cuda.driver.major":  driverMajor,
 		"nvidia.com/cuda.driver.minor":  driverMinor,
 		"nvidia.com/cuda.driver.rev":    driverRev,
 		"nvidia.com/cuda.runtime.major": fmt.Sprintf("%d", *cudaMajor),
 		"nvidia.com/cuda.runtime.minor": fmt.Sprintf("%d", *cudaMinor),
```

Comment:
My one question was whether we wanted the following set of labels instead:
```
		"nvidia.com/cuda.driver.major"
		"nvidia.com/cuda.driver.minor"
		"nvidia.com/cuda.driver.rev"
		"nvidia.com/cuda.driver.revision"
		"nvidia.com/cuda.driver.full"
		"nvidia.com/cuda.runtime.major"
		"nvidia.com/cuda.runtime.minor"
		"nvidia.com/cuda.runtime.revision"
		"nvidia.com/cuda.runtime.full"
```
So as to not have to deprecate labels except for `"nvidia.com/cuda.driver.rev"`?

@klueska do you have thoughts w.r.t the `-version` string inserted above?

Reply:
I have a slight preference towards `driver-version` and `runtime-version` as it is more explicit.